govcd/externalnetwork_test.go

```diff
@@ -193,6 +191,11 @@ func (vcd *TestVCD) Test_CreateExternalNetwork(check *C) {
 	check.Assert(ipRange.EndAddress, Equals, "192.168.201.250")
 
 	check.Assert(newExternalNetwork.ExternalNetwork.Configuration.FenceMode, Equals, "isolated")
+	check.Assert(newExternalNetwork.ExternalNetwork.Description, Equals, "Test Create External Network")
```

Comment:
@dataclouder Do I understand correctly that this PR is trying to solve an issue you hit where a change through UI in description of external network doesn't get noticed by Terraform? If yes:
1. Can you confirm that code from this PR solved it?
2. I believe we need a test case here which checks that after updating description it gets read correctly. I.e. not only that it is set correctly (which works even without this PR).

All in all, I'm asking for a test case which tests what the PR is fixing.

Reply:
No. The issue was that, using the previous data structure, some fields in the external network were not converted to the Go structure when retrieved.
The creation was correct: it was setting everything the way we expected, but when retrieving the data, fields like description and vcenter details were not retrieved, although the external network had them set correctly.

Without this fix, neither data source or importing would work on the Terraform side.

There is already a test that checks for all these details. I can add an extra test and propose it for inclusion in this PR on Monday morning.

Reply:
Upon further inspection, I see that the existing test already checks for everything we need.

Reply:
Does this PR fix the issue where a change through UI in description of external network doesn't get read by Terraform? If yes, can we add a test case which checks whether such background change gets read afterwards correctly?

Reply:
The PR fix the issue that was reported. 
The problem was that description was created correctly, but never retrieved, regardless of whether it was modified using API or UI.

Reply:
Synced with Vaidotas. As external network doesn't have update function yet, the test I was asking for is not timely to implement yet. On the other hand, he confirmed that the new asserts that are added below fail with old code and succeed with new code, so it's enough for now.